src/constructors/css.js

```diff
@@ -4,6 +4,11 @@ import flatten from '../utils/flatten'
 import type { Interpolation, RuleSet } from '../types'
 
 export default (
-  strings: Array<string>,
+  strings: Array<string> | Object,
```

Comment:
We should probably rename this property. Perhaps "styles"?

Reply:
Yeah that makes sense, incoming.